deposits.js

```diff
@@ -0,0 +1,6 @@
+  function getDetails(){
+    let dname = document.getElementById("name");
```

Comment:
A few questions:
1. What is the value of `dname`?
2. Why is it called `dname`?

Reply:
dname refers to the name entered by the person depositing. I have not added any value to it.

Reply:
Can you console log the value after adding it and paste it here?

Why not just call it `name` ? 🤔

Reply:
it is  a bit confusing considering the id on the slim file is also name. i had to distinguish  one or the other

Reply:
Okay great